sites/docs/src/pages/docs/index.astro

```diff
@@ -10,7 +10,15 @@ let docs = await getCollection('docs');
 import SidebarNav from '@components/sidebar/SidebarNav.astro';
 
 let sections: SidebarEntry[] = [
-    { type: 'group', id: 'usage', label: 'Usage', entries: [], collapsed: true, icon: 'octicon:play-16', weight: 1 },
+    {
+        type: 'group',
+        id: 'guides',
+        label: 'Guides',
```

Comment:
I don't get it - the PR preview still shows this as Guides I think..? In fact it looks exactly like the live site as far as I can tell. Is the preview rendering correctly @christopher-hakkaart ?

Reply:
It shows locally - I'll look into it

Reply:
I guess this is similar issues to what we've seen elsewhere with deployment previews after the new multi-site split thing. 

I think we may need to write some custom GitHub actions code to generate previews..